src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,122 @@
+package duke;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+/**
+ * Contains the task list which contains the tasks created by the user.
+ */
+public class TaskList {
+
+    ArrayList<Task> tasks;
```

Comment:
Consider using List<Task> instead, in case you want to use a different implementation of List in the future.